src/enhanced_transactions.rs

```diff
@@ -31,14 +31,20 @@ impl Helius {
     ///
     /// # Arguments
     /// * `address` - An address for which a given parsed transaction history will be retrieved
```

Comment:
We'll need to update this to reflect the new request type as the `address` argument will no longer be valid

Reply:
parse_transactions follows this format as well (showing the parameters of the type) which is why I framed it that way. Would you like me to update that one as welll?

Reply:
As in, I put `address` and `before` instead of `ParsedTransactionHistoryRequest', like parse_transactions does above 
<img width="508" alt="CleanShot 2024-05-14 at 08 11 14@2x" src="https://github.com/helius-labs/helius-rust-sdk/assets/104528437/7a46a73b-738b-407a-9989-c7fb1d8b1d13">
<img width="508" alt="CleanShot 2024-05-14 at 08 11 14@2x" src="https://github.com/helius-labs/helius-rust-sdk/assets/104528437/7a46a73b-738b-407a-9989-c7fb1d8b1d13">
<img width="508" alt="CleanShot 2024-05-14 at 08 11 14@2x" src="https://github.com/helius-labs/helius-rust-sdk/assets/104528437/7a46a73b-738b-407a-9989-c7fb1d8b1d13">

Reply:
Ah, yeah, that's an oversight on my end — `parse_transactions` should have `ParseTransactionsRequest` as its argument and the description should stay the same